core/selection.cpp

```diff
@@ -231,3 +237,19 @@ extern "C" void select_newest_visible_dive()
 	// No visible dive -> deselect all
 	select_single_dive(nullptr);
 }
+
+extern "C" void select_trip(struct dive_trip *trip)
```

Comment:
so in this commit I only see a call to `deselect_trip()` and no calls to `select_trip()` which makes me wonder... but most likely that's simply in the next commits?